source/tutorials/index.rst

```diff
@@ -3,9 +3,56 @@
 Tutorials
 #########
 
-Explore our tutorials to discover what you can do with |CL|!
+Explore our tutorials to discover what you can do with |CL|! Now conveniently
+sorted by experience level!
```

Comment:
Hey @intelkevinputnam. Let's discuss a standard word in place of "experience level"--based on our emails about this. 
fyi: @bktan8, @krisdale

Whatever we decide on for "experience level," that phrase can be also be a hyperlink to a new document
where we explain the rating levels.

Reply:
Low, Moderate, and High are not descriptions for "experience level".  If we're going with levels, they should be "Beginner, Immediate, Advanced"?

Reply:
Easy, Moderate, Difficult.  Nice. Thanks, all.